test/helpers/kubectl.go

```diff
@@ -3405,6 +3405,35 @@ func (kub *Kubectl) WaitForIPCacheEntry(node, ipAddr string) error {
 		&TimeoutConfig{Timeout: HelperTimeout})
 }
 
+// RepeatCommandInBackground runs command on repeat in goroutine until quit channel
+// is closed and closes run channel when command is first run
+func (kub *Kubectl) RepeatCommandInBackground(cmd string) (quit, run chan struct{}) {
+	quit = make(chan struct{})
+	run = make(chan struct{})
+	go func() {
+		firstRun := true
+	loop:
+		for {
+			select {
+			case <-quit:
+				break loop
```

Comment:
Don't you mean `return` here?

Reply:
Sure, good catch, will get rid of the label too.